pkg/api/vlabs/validate_test.go

```diff
@@ -1083,6 +1083,18 @@ func Test_Properties_ValidateContainerRuntime(t *testing.T) {
 		)
 	}
 
+	p.OrchestratorProfile.KubernetesConfig.ContainerRuntime = "kata-containers"
```

Comment:
Is kata-containers supported for all k8s versions?

Reply:
Kata is more tightly couple with the CRI-shim version (in this case, containerd).  I think if there's an error, it'll likely be a mismatch between containerd + k8s?